src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -182,6 +188,25 @@ public SSH(final String adr, final int prt,
     }
 
     /**
+     * Constructor.
+     * @param adr IP address
+     * @param prt Port of server
+     * @param user Login
+     * @param priv Private SSH key
+     * @param passphrs Pass phrase for encrypted priv. key
+     * @throws UnknownHostException when host is unknown.
+     * @checkstyle ParameterNumberCheck (6 lines)
+     */
+    public SSH(final String adr, final int prt,
```

Comment:
@LinkedList what happened with the constructor we had before? just deleted it? not a good idea. we need to have it, but make it secondary...